src/Components/HabitTracker/HabitItem.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+
+const TrackerButton = ({ title, duration }) => {
```

Comment:
I think here instead of the duration:
we can have an array of object 
the object can be {id: int, status: string}
the status can define if the day is checked (green), unchecked(red), or notYet (grey) 
and onClick will change the status 
because we need to save this information somewhere later